File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.js
// Last updated: 3/6/2026, 2:08:48 PM
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let largest = 0;
    for(let i=0;i<accounts.length;i++){
        let sum = 0;
        for(let j=0;j<accounts[i].length;j++){
           sum = sum + accounts[i][j];
           if(accounts[j] > largest){
            largest = accounts[j];
           }
           if(sum > largest){
            largest = sum;
           }
        }
    }
    return largest
};

File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.cpp
// Last updated: 7/17/2026, 4:26:06 PM
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int largest = 0;

        for (int i = 0; i < accounts.size(); i++) {
            int sum = 0;

            for (int j = 0; j < accounts[i].size(); j++) {
                sum += accounts[i][j];

                if (sum > largest) {
                    largest = sum;
                }
            }
        }

        return largest;
    }
};